### PR TITLE
unmanaged detections tuning

### PR DESCRIPTION
**Pull request by calkim-panther**
### Background

Tuning unmanaged detections

### CODE REVIEW

File: rules/crowdstrike_rules/onepassword_login_from_crowdstrike_unmanaged_device.py [hunk 1]
```diff
@@ -9,5 +9,5 @@ def title(event):
     return (
         "1Password Login for "
         f"[{deep_get(event, 'target_user', 'email', default = '<email_not_found>')}]"
-        " from unmanaged device."
+        " from unmanaged IP Address."
```
Comment (edyesed) [inline rules/crowdstrike_rules/onepassword_login_from_crowdstrike_unmanaged_device.py:12]: I'm a little unclear about why this language change. I don't immediately understand how IP Addresses are managed things in crowdstrike

**Review by edyesed — COMMENTED** [review]
question about title verbiage change

File: rules/crowdstrike_rules/onepassword_login_from_crowdstrike_unmanaged_device.py (hunk 1 above)
Comment (calkim-panther) [reply rules/crowdstrike_rules/onepassword_login_from_crowdstrike_unmanaged_device.py:12]: I'm using IP addresses found in Crowdstrike's AIP list to compare to. Technically, the detection wouldn't catch if someone logged in from a different device but from the same IP so I wanted to make that more clear.

**Review by edyesed — APPROVED** [review]
tunes some crowdstrike queries to run hourly and updates their alerts texts to clarify that they detect based on IP address vs device.

**Pull request merged**